src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/appointments/AppointmentsResource.kt

```diff
@@ -0,0 +1,69 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.appointments
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import jakarta.validation.Valid
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.validation.annotation.Validated
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.config.ErrorResponse
+
+@RestController
+@Validated
+@RequestMapping(produces = [MediaType.APPLICATION_JSON_VALUE])
+class AppointmentsResource(private val appointmentService: AppointmentService) {
+  @PreAuthorize("hasRole('ROLE_NOMIS_APPOINTMENTS')")
+  @PostMapping("/appointments")
```

Comment:
Not sure it matters too ,much but do you think this should be 
`/prisoners/booking-id/{bookingId}/appointments`  so it is consistent with IEPs and sentencing adjustments?

Reply:
Actually I think appointments are conceptually a bit different because they are transitory - i.e. the event happens, then it expires. So the concept of a prisoner having a 'list' of appointments is not really important.